### PR TITLE
selectize.js Integration

### PR DESCRIPTION
**Pull request by klopfdreh**
See examples:

https://github.com/klopfdreh/core/blob/435e7aa23f6e45daeb7b0b97ad7da7a2bd8571b2/wicketstuff-selectize-parent/wicketstuff-selectize-examples/src/main/java/org/wicketstuff/selectize/TestPage.java

### CODE REVIEW

**Comment by klopfdreh** [comment]
If everything is ok - we have to add the module as submodule.

File: wicketstuff-selectize-parent/wicketstuff-selectize-examples/src/main/java/org/wicketstuff/selectize/TestPage.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+		// Select with ajax
+		Selectize selectize4 = new Selectize("selectize4")
+		{
+			private static final long serialVersionUID = 1L;
+
+			@Override
+			protected SelectizeResponse response(String search)
+			{
+				if (search != null && search.equals("test"))
```
Comment (martin-g) [inline wicketstuff-selectize-parent/wicketstuff-selectize-examples/src/main/java/org/wicketstuff/selectize/TestPage.java]: Could be simplified with: `if ("test".equals(search))`

**Comment by martin-g** [comment]
:+1: 
Just mention somewhere in the javadoc that the template should be Handlebars.

**Comment by klopfdreh** [comment]
I am going to write some javadoc tomorrow evening and after that I think we can integrate it.

**Pull request merged**